chompfile.toml

```
@@ -43,6 +43,27 @@ target = 'dist/lexer.cjs'
 deps = ['dist/lexer.js']
 run = 'babel dist/lexer.js | terser -c -m -o dist/lexer.cjs'
 
+[[task]]
+name = 'build:swc'
+target = 'src/lexer.js'
+dep = 'src/lexer.ts'
+# Note we should use the chomp swc template, but
+# https://github.com/swc-project/cli/issues/113 means we always get a sourcemap
+# even when we set "source-maps = false", so for now we have ejected the
+# template to its raw "run" command, and added an "rm" step.
+run = 'node ./node_modules/@swc/cli/bin/swc.js $DEP -o $TARGET --no-swcrc -C jsc.parser.syntax=typescript -C jsc.parser.importAssertions=true -C jsc.parser.topLevelAwait=true -C jsc.parser.importMeta=true -C jsc.parser.privateMethod=true -C jsc.parser.dynamicImport=true -C jsc.target=es2016 -C jsc.experimental.keepImportAssertions=true && rm src/lexer.js.map'
```

One of the main reasons I worked on Chomp is because I run windows and got really annoyed with the lack of powershell support. Unfortunately it doesn't support `&&` and `||` just yet, so would be great to stick to multiline for now:

```suggestion
run = '''
node ./node_modules/@swc/cli/bin/swc.js $DEP -o $TARGET --no-swcrc -C jsc.parser.syntax=typescript -C jsc.parser.importAssertions=true -C jsc.parser.topLevelAwait=true -C jsc.parser.importMeta=true -C jsc.parser.privateMethod=true -C jsc.parser.dynamicImport=true -C jsc.target=es2016 -C jsc.experimental.keepImportAssertions=true
rm src/lexer.js.map
'''
```